apps/omg_api/test/monitor_test.exs

```diff
@@ -86,12 +89,14 @@ defmodule OMG.API.MonitorTest do
   test "if a child gets a interval set after termination" do
     child = {__MODULE__.Mock, []}
     {:ok, monitor_pid} = Monitor.start_link([child])
-    Process.unlink(monitor_pid)
-    Alarm.raise({:ethereum_client_connection, :erlang.node(), __MODULE__})
-    spawn(fn -> __MODULE__.Mock.terminate(:ethereum_client_connection) end)
+    _ = Process.unlink(monitor_pid)
+    _ = Alarm.raise({:ethereum_client_connection, :erlang.node(), __MODULE__})
+    _ = spawn(fn -> __MODULE__.Mock.terminate(:ethereum_client_connection) end)
     assert pull_state_and_find_timer(monitor_pid, 1000)
   end
 
+  @tag :capture_log
+  @tag :integration
```

Comment:
Making entire module (all test) as integration and moving file to ‚/integration’ would be cleaner for me.

Reply:
Hm... This test isn't really _integration_ it's only tagged this way because it's slow and we're not able to parallel execute coveralls (_not yet at least_). So marking all of them is... I don't think it makes a difference at this point.